### ISSUE TITLE
Statistics Time Range Filter Not Working

### ISSUE THREAD

**Issue by melink14** [dermow/TraXile#95]
Hi again!

The time filtering with league presets is a great idea (also love the stopwatch overlay) but it doesn't seem to work for me.

Any non-all preset I select shows no data despite being able to see data on the tracking tab as well as fresh data in the EventLog.

If I select 'all' then I can see what seems to be accurate data except that 'hideout' time is currently 0 even though I spend a lot of time in my hideout (and it used to show non 0 before the refactor!)

Let me know if there's some more information I can provide nothing related to rendering the dashboard showed up in the logs but I can help you debug with test/screenshots etc.

**Comment by melink14** [comment]
(This may be related to https://github.com/dermow/TraXile/issues/92 since it seems my stored timestamps are messed up)

**Comment by dermow** [comment]
Hi! 

Agree, that the source issue is #92 - Could you provide me your Client.txt file? 

If you do not want to post it here, you can reach me on discord: mowgli1988#1400

Thanks for your help!

**Comment by melink14** [comment]
I'll pass it through discord since it's easier than considering the contents. :)

**Comment by melink14** [comment]
(It seems that discord doesn't allow messaging new people directly so I plan to temporarily add you as a friend from melink14#5735 unless there's a better way)

**Comment by melink14** [comment]
Though maybe this snippet is enough?
```
2021/12/28 16:13:15 832858015 124 [INFO Client 40596] Connecting to instance server at 165.192.83.87:6112
2021/12/28 16:13:15 832858031 1ca [DEBUG Client 40596] Connect time to instance server was 16ms
2021/12/28 16:13:15 832858062 412 [DEBUG Client 40596] Generating level 67 area "2_9_town" with seed 1
2021/12/28 16:13:15 832858421 8e [INFO Client 40596] Tile hash: 3828770056
2021/12/28 16:13:15 832858421 8f [INFO Client 40596] Doodad hash: 0
2021/12/28 16:13:16 832859093 9ac [DEBUG Client 40596] Joined guild named Elanus Risk Control with 15 members 
2021/12/28 16:13:16 832859296 b87 [INFO Client 40596] : You have entered Highgate.
2021/12/28 16:13:17 832860281 52d [INFO Client 40596] [SHADER] Delay: ON
2021/12/28 16:13:21 832864812 d63 [DEBUG Client 40596] Got Instance Details from login server
2021/12/28 16:13:21 832864812 52d [INFO Client 40596] [SHADER] Delay: OFF
2021/12/28 16:13:21 832864812 124 [INFO Client 40596] Connecting to instance server at 165.192.83.93:6112
2021/12/28 16:13:21 832864843 1ca [DEBUG Client 40596] Connect time to instance server was 16ms
2021/12/28 16:13:22 832864859 412 [DEBUG Client 40596] Generating level 60 area "HideoutBeach" with seed 1
2021/12/28 16:13:22 832864906 8e [INFO Client 40596] Tile hash: 2449035884
2021/12/28 16:13:22 832864906 8f [INFO Client 40596] Doodad hash: 0
2021/12/28 16:13:22 832865593 9ac [DEBUG Client 40596] Joined guild named Elanus Risk Control with 15 members 
2021/12/28 16:13:22 832865750 b87 [INFO Client 40596] : You have entered Coastal Hideout.
2021/12/28 16:13:23 832866578 52d [INFO Client 40596] [SHADER] Delay: ON
2021/12/28 16:28:28 833771562 b87 [INFO Client 40596] : AFK mode is now ON. Autoreply "This player is AFK."
2021/12/28 16:43:46 834688953 b87 [INFO Client 40596] : AFK mode is now OFF.
2021/12/28 17:05:20 835983343 b87 [INFO Client 40596] : AFK mode is now ON. Autoreply "This player is AFK."
2021/12/28 17:07:20 836103453 b87 [INFO Client 40596] : AFK mode is now OFF.
2021/12/28 17:22:21 837004640 b87 [INFO Client 40596] : AFK mode is now ON. Autoreply "This player is AFK."
2021/12/28 18:27:20 840903234 b87 [INFO Client 40596] : AFK mode is now OFF.
2021/12/28 18:42:21 841803968 b87 [INFO Client 40596] : AFK mode is now ON. Autoreply "This player is AFK."
```

**Comment by melink14** [comment]
(sent some more via discord)

**Comment by dermow** [comment]
As worked out on discord, solution was to enforce the gregorian calendar in the Source code. Thanks again for reporting. Will be fixed with the next release (0.9.1)

**Comment by dermow** [comment]
Fixed with 0.9.1
